tox.ini

```diff
@@ -42,8 +42,6 @@ commands =
 
 [travis]
 python =
-  2.7: py27, style
```

Comment:
Do we want to run the `style` commands under a different Python env?

Reply:
We already are for 3.8.

Reply:
lol didn't see it.